### DISCUSSION TITLE
Compile is slow

### DISCUSSION THREAD

**Discussion by Taxi4you** [vercel/next.js#69754] [Ideas]
### Goals

Make it faster please.

### Background

Any hardware or OS, the platform compilation is slow.

### Proposal

DEVELOPER EXPERIENCE IS MORE IMPORTANT THAN ANYTHING

**Comment by ghost** [comment]
javascript is supposed to be interpreted, what does it even compile for so long lmao

**Comment by ghost** [comment]
probably just their spywar.. ahem telemetry
